mindsdb/api/http/namespaces/handlers.py

```diff
@@ -112,6 +112,48 @@ def post(self, handler_name):
         )
 
 
+@ns_conf.route('/<handler_name>/test_connection')
```

Comment:
Maybe we choose a better path name as `health` or `status` ? WDYT?

Reply:
Hmm.. Would `connection/status` be better? Just to avoid any confusion between the status of the connection and the handler itself?

Reply:
I was thinking of something to sound more like action instead of using a verb in path

Reply:
OK, shall I make it `handler/status` then?

Reply:
Yes, please also add docs